code/modules/mob/living/carbon/human/say.dm

```diff
@@ -167,6 +167,12 @@
 				S.message = uppertext(S.message)
 				verb = "yells loudly"
 
+		if(istype(wear_mask, /obj/item/clothing/mask/gas/voice_modulator))
```

Comment:
The way masks are handled here could do with a refactor. Out of the scope for your PR though I recon